### PR TITLE
docs: change the file name of the public API according with Angul…

### PR DESCRIPTION
**Pull request by wKoza**
…ar CLI

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

A fix in the documentation

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Currently, the documentation says that the file name of the public API is index.ts but it should be public_api.ts

## What is the new behavior?
it's fixed

### CODE REVIEW

File: aio/content/guide/creating-libraries.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This builder, among other things, ensures that the library is always built with
 
 To make library code reusable you must define a public API for it. This "user layer" defines what is available to consumers of your library. A user of your library should be able to access public functionality (such as NgModules, service providers and general utility functions) through a single import path.
 
-The public API for your library is maintained in the `index.ts` file of your library folder.
+The public API for your library is maintained in the `public_api.ts` file of your library folder.
```
Comment (gkalpak) [inline aio/content/guide/creating-libraries.md]: Where does this file come from? Does the cli generate that?
The name seems inconsistent with the kebab-case convention used in other files.

**Review by gkalpak — CHANGES_REQUESTED** [review]
Please, change the commit message to `docs: ...`

File: aio/content/guide/creating-libraries.md (hunk 1 above)
Comment (wKoza) [reply aio/content/guide/creating-libraries.md]: It's right, it's inconsistent. Yes, this file come from `ng generate library`. Should I do the change also in Angular CLI ?

Comment (gkalpak) [reply aio/content/guide/creating-libraries.md]: 😱 There should at least be a discussion about that. Can you open an issue in cli (and cc me)?

Comment (wKoza) [reply aio/content/guide/creating-libraries.md]: Ok, There are not many people on the dedicated Slack :sleeping:

Comment (alan-agius4) [reply aio/content/guide/creating-libraries.md]: I agree with @gkalpak, should should follow the convention and be kebab case.

Comment (wKoza) [reply aio/content/guide/creating-libraries.md]: @alan-agius4, I've pushed https://github.com/angular/angular-cli/pull/13866. So, we have to be synchronized with this PR. 
@gkalpak , the commit message has been changed.

File: aio/content/guide/creating-libraries.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ This builder, among other things, ensures that the library is always built with
 
 To make library code reusable you must define a public API for it. This "user layer" defines what is available to consumers of your library. A user of your library should be able to access public functionality (such as NgModules, service providers and general utility functions) through a single import path.
 
-The public API for your library is maintained in the `index.ts` file of your library folder.
+The public API for your library is maintained in the `public-api.ts` file of your library folder.
```
Comment (alan-agius4) [inline aio/content/guide/creating-libraries.md]: Nit: _file of your -> file in your_

**Review by alan-agius4 — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]
Thx, @wKoza 👍
LGTM as soon as angular/angular-cli#13866 is merged.

**Comment by alan-agius4** [comment]
CLI PR has been merged

**Review by gkalpak — APPROVED** [review]

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**